src/app/landmarks/DlibXML.h

```diff
@@ -72,8 +72,8 @@ inline cv::Size read_png_size(const std::string& filename)
     {
         unsigned int width, height;
         in.seekg(16);
-        in.read(reinterpret_cast<char*>(&width), 4);
-        in.read(reinterpret_cast<char*>(&height), 4);
+        in.read(reinterpret_cast<char*>(&width), 4); // NOLINT (TODO)
+        in.read(reinterpret_cast<char*>(&height), 4); // NOLINT (TODO)
```

Comment:
So basically every `reinterpret_cast<>` is flagged by `clang-tidy` even if the interface may require it in some cases http://www.cplusplus.com/reference/istream/istream/read/ `istream& read (char* s, streamsize n);`?  Does `clang-tidy` suggest an alternative?